chapters/operatorsandexpressions.tex

```diff
@@ -1401,36 +1401,54 @@ \subsection{Constant Expressions}\label{constant-expressions}
 
 Components declared as \lstinline!constant! shall have an associated declaration equation with a constant expression, if the constant is directly in the simulation model, or used in the simulation model.  The value of a constant can be modified after it has been given a value, unless the constant is declared \lstinline!final! or modified with a \lstinline!final! modifier.  A constant without an associated declaration equation can be given one by using a modifier.
 
-\subsection{Parameter Expressions}\label{parameter-expressions}
+\subsection{Structural Expressions}\label{structural-expressions}
 
-Parameter expressions\index{parameter expression}\index{expression variability!parameter}\index{parametric variability|see{parameter expression}} are:
+Structural expressions\index{structural expression}\index{expression variability!structural} are:
 \begin{itemize}
 \item
   Constant expressions.
 \item
-  Variables declared as \lstinline!parameter!.
-\item
-  Input variables in functions behave as though they were parameter expressions.
+  Structural parameters (\cref{component-variability-prefixes-discrete-parameter-constant}).
 \item
-  Except for the special built-in operators \lstinline!initial!, \lstinline!terminal!, \lstinline!der!,
-  \lstinline!edge!, \lstinline!change!, \lstinline!sample!, and \lstinline!pre!, a function or operator with parameter
-  subexpressions is a parameter expression.
+  Except for the special built-in operators \lstinline!initial!, \lstinline!terminal!, \lstinline!der!, \lstinline!edge!, \lstinline!change!, \lstinline!sample!, and \lstinline!pre!, a function or operator with structural subexpressions is a structural expression.
 \item
-  Some function calls are parameter expressions even if the arguments are not:
+  Some function calls are structural expressions even if the arguments are not:
   \begin{itemize}
   \item
     \lstinline!cardinality(c)!, see restrictions for use in \cref{cardinality-deprecated}.
   \item
     \lstinline!end! in \lstinline!A[$\ldots$ end $\ldots$]! if \lstinline!A! is variable declared in a non-\lstinline!function! class.
   \item
-    \lstinline!size(A)! (including \lstinline!size(A, j)! where \lstinline!j! is parameter expression) if \lstinline!A! is variable declared in a non-function class.
+    \lstinline!size(A)! (including \lstinline!size(A, j)! where \lstinline!j! is a structural expression) if \lstinline!A! is variable declared in a non-function class.
   \item
     \lstinline!Connections.isRoot(A.R)!
   \item
     \lstinline!Connections.rooted(A.R)!
   \end{itemize}
 \end{itemize}
 
+\subsection{Parameter Expressions}\label{parameter-expressions}
```

Comment:
I don't know if it is within the scope of this PR but this section could include "expressions in the body of initial equation, or initial algorithm."  instead of the discrete-time section.

Reply:
Yes, wouldn't this fit better in #2602?

Reply:
Yes if this PR is approved I will propose the relevnt change in #2602

Reply:
Good.  Then I think this can be considered resolved.